browser/ui/views/sidebar/sidebar_control_view.h

```diff
@@ -85,6 +93,7 @@ class SidebarControlView : public views::View,
   void UpdateSettingsButtonState();
   void UpdateBackgroundAndBorder();
 
+  raw_ptr<Delegate> delegate_;
```

Comment:
nit: for consistency

```suggestion
  raw_ptr<Delegate> delegate_ = nullptr;
```